v3/position_test.go

```diff
@@ -1,17 +1,27 @@
 package geohex
 
 import (
+	"fmt"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("Position", func() {
 
 	It("should generate centroids", func() {
-		pos := &Position{4, -5, zooms[0]}
+		pos := &Position{X: 4, Y: -5, z: zooms[0]}
 		cnt := pos.Centroid()
 		Expect(cnt.E).To(BeNumerically("~", 20037508.3, 0.1))
 		Expect(cnt.N).To(BeNumerically("~", -1285406.8, 0.1))
 	})
 
+	for _, tc := range loadPosition2HexTestCases() {
```

Comment:
sorry, but could you pls change this to use https://onsi.github.io/ginkgo/#table-driven-tests